openapi/p4pa-process-executions.openapi.yaml

```
@@ -109,6 +124,31 @@ components:
           type: string
         fileOrigin:
           type: string
+    ExportFileRequestDTO:
+      type: object
+      required:
+        - organizationId
+        - filePathName
+        - fileName
+        - fileSize
+        - flowFileType
+      properties:
+        organizationId:
+          type: integer
+          format: int64
+        filePathName:
+          type: string
+        fileName:
+          type: string
+        fileSize:
+          type: integer
+          format: int64
+        flowFileType:
+          type: string
+          enum:
+            - PAID
+            - CLASSIFICATIONS
+            - PAYMENTS_REPORTING
```

Use typeMapping to re-use the already define enum
```suggestion
          type: 'ExportFlowFileType'
```